README.md

```diff
@@ -4,29 +4,288 @@ See documentation [here](doc/00-overview.md)
 
 ``` yaml
 use-extension:
-  "@autorest/modelerfour": "~4.4.162"
-
-try-require: ./readme.cli.md
+  "@autorest/modelerfour": "4.6.200"

```

Comment:
Is it a must need of using clicommon modifiers with M4 version 4.6.200 ? because currently autorest.python that az is using is 4.4.162. if yes, we probably need to ask Laurent to upgrade the modelerfour in autorest.python

Reply:
just did a quick check, seems latest autorest.python is using 4.8.221 already... could you confirm?

Reply:
ok, I will confirm with Laurent about that. But Garrett says. in modelerfour 4.7.+, they use multiple requests in one single operation definition to solve different media type request. I am afraid it may cause breakingChanges.

Reply:
yes, I noticed that too, so plan to have this merged first and then try to bump the autorest and fix any breaking changes from that.